app/views/devise/unlocks/new.html.erb

```diff
@@ -1,12 +1,20 @@
-<h2>Resend unlock instructions</h2>
+<div class="redtop">
+  <div class="container">
+    <div class="row">
+      <div class="resend-unlock col-lg-4 col-lg-offset-4 col-sm-6 col-sm-offset-3">
+        <h2>Resend unlock instructions</h2>
 
-<%= form_for(resource, :as => resource_name, :url => unlock_path(resource_name), :html => { :method => :post }) do |f| %>
-  <%= devise_error_messages! %>
+        <%= form_for(resource, :as => resource_name, :url => unlock_path(resource_name), :html => { :method => :post }) do |f| %>
+          <%= devise_error_messages! %>
 
-  <div><%= f.label :email %><br />
-  <%= f.email_field :email, :autofocus => true %></div>
+          <div><%= f.label :email %><br />
```

Comment:
Add `class="form-group"` to the `<div>` and `class: "form-control"` as an option to the email field.